src/main.rs

```diff
@@ -42,6 +46,7 @@ struct VerificationKey {
     verification_key: String,
 }
 
+#[allow(dead_code)]
```

Comment:
This directive prevents lint (clippy) warnings about un-read values.

Reply:
I propose an alternate way of handling these warnings; by actually removing the offending fields that are unread.  

What is going on here is that the `SindriTaskStatusResponse` definition needs to include types for keys that we will be extracting from an API response (so that the `#[derive(Deserialize)]` macro can do its thing) but we don't have to instruct the deserializer how to handle any field we don't care about.  For that reason, I'd propose you remove:
```
    pub project_name: String,
    pub perform_verify: bool,
    pub queue_time_sec: Option<f64>,
    pub public: Option<serde_json::Value>,
    pub warnings: Option<Vec<String>>,
```
The onus of adding them back in will be on the person that actually wants to use them in this codebase

Reply:
Fields removed and struct renamed in [cb58816](https://github.com/Sindri-Labs/sindri-scroll-sdk/pull/17/commits/cb588166d2aa383c68e2130b54424f63cfeaf9e5).